trogdor/src/main/java/org/apache/kafka/trogdor/rest/JsonRestServer.java

```diff
@@ -138,13 +138,13 @@ public void beginShutdown() {
                     log.info("Stopping REST server");
                     jettyServer.stop();
                     jettyServer.join();
+                    jettyServer.destroy();
                     log.info("REST server stopped");
                 } catch (Exception e) {
                     log.error("Unable to stop REST server", e);
                 } finally {
-                    jettyServer.destroy();
+                    shutdownExecutor.shutdown();
```

Comment:
Shouldn't jetty.destroy remain in the finally and maybe we catch any exceptions it throws?